activerecord/test/schema/schema.rb

```diff
@@ -310,6 +310,11 @@ def except(adapter_names_to_exclude)
   create_table :eyes, force: true do |t|
   end
 
+  create_table :flowers, force: true do |t|
```

Comment:
There seem to be a few different ways to create a temporary table used in the other tests but I've now gone with directly using `ActiveRecord::Base.connection.create_table` as the model is actually unnecessary.